internal/slice/slice.go

```diff
@@ -0,0 +1,16 @@
+package slice
+
+const (
+	Wildcard    = "*"
+	X509Subject = "x509.subject"
+)
+
+// Contains is a utility function to check if a string exists in an array
+func Contains(val string, values []string) bool {
```

Comment:
Can we have something like https://pkg.go.dev/golang.org/x/exp/slices#Contains?

Reply:
The reason why we don't directly import `golang.org/x/exp` is that that repo is not stable. Here's the statement from `exp` package:
> **Warning**: Packages here are experimental and unreliable. Some may one day be promoted to the main repository or other subrepository, or they may be modified arbitrarily or even disappear altogether.

Reply:
I tried this before, but []string does not implement comparable.